llama_stack/distribution/datatypes.py

```diff
@@ -253,6 +264,10 @@ class ServerConfig(BaseModel):
         default=None,
         description="Authentication configuration for the server",
     )
+    quota: QuotaConfig | None = Field(
+        default=None,
+        description=("Per client request quota configuration. If unset or null, quotas are disabled."),
```

Comment:
```suggestion
        description=("Per client request quota configuration."),
```

I think it's obvious that if unset quota are disabled :)